configs/product/power.mk

```diff
@@ -20,6 +20,4 @@
 #####################
 
 PRODUCT_PACKAGES += \
-	power.universal7580 \
-	android.hardware.power@1.0-service \
-	android.hardware.power@1.0-impl
+	android.hardware.power@1.0-service.universal7580
```

Comment:
So you are deleting all the custom power HAL files but enabling building the module here?
With this change you will not be able to build the ROM.

Reply:
You don't need these files. It's possible without them

Reply:
https://github.com/Exynos7580/android_device_samsung_s5neoltexx/commit/106678c0e74298d9e384d48943bd6f169246a61a

Reply:
That only works because @Stricted created a new HIDL based power HAL in the lineage-16.0 branch of android_hardware_samsung. This will not work in the lineage-15.1 branch.

Reply:
k i removed it now